runtime/calamari/src/weights/pallet_utility.rs

```diff
@@ -56,49 +56,49 @@ pub trait WeightInfo {
 pub struct SubstrateWeight<T>(PhantomData<T>);
 impl<T: frame_system::Config> pallet_utility::WeightInfo for SubstrateWeight<T> {
     fn batch(c: u32, ) -> Weight {
-        (15_467_000 as Weight)
-            // Standard Error: 3_000
-            .saturating_add((4_037_000 as Weight).saturating_mul(c as Weight))
+        (28_988_000 as Weight)
```

Comment:
that's a 2x

Reply:
yeah, 2x will affect how many normal `batch` can be inserted a block, roughly 1/2
```
RuntimeBlockWeights * 0.7 / (28_988_000 + 4_490_000) * 1
```

Reply:
Could be within error bounds of benchmarking

Most of the weight is included in the `.saturating_add((4_490_000 as Weight).saturating_mul(c as Weight))` which can make the base weight have bad precision.

+1 to running again and seeing if it reproduces similar weight

Reply:
Just triggered our CI.

Reply:
benchmark rerun comparison, no significant change
![image](https://user-images.githubusercontent.com/4237655/210402402-ba821484-d965-443f-a422-49ef66297e3e.png)

Reply:
IMO this is wholly irrelevant tho, because batch calls don't get inserted en-masse in an empty state. This measures the overhead of running a TX through batch instead of standalone and so the actual TX payload is likely to be orders of magnitude heavier than this batch execution overhead weight.A single `to_private` is 10x this amount, so I don't really care that this got 2x slower

Reply:
what was the result of https://github.com/Manta-Network/Manta/actions/runs/3829937963/attempts/1

Reply:
see screenshot above on the right

Reply:
ok, nice missed that.